src/ResourceManager/DataLakeStore/Commands.DataLakeStore/help/Move-AzureRmDataLakeStoreItem.md

```diff
@@ -49,9 +49,7 @@ Accept wildcard characters: False
 ```
 
 ### -DefaultProfile
-The credentials, account, tenant, and subscription used for communication with azure
-
-```yaml
+The credentials, account, tenant, and subscription used for communication with azure.```yaml
```

Comment:
@rahuldutta90 this is a bug in the `platyPS` module - please make sure that you move the ` ```yaml ` string at the end of this line to a line by itself. You can see how this should be done by looking at any other parameter in this file.

Reply:
Good catch. This one I missed.